assets/javascripts/discourse/templates/components/topic-chat-float.hbs

```diff
@@ -38,31 +38,28 @@
         {{/if}}
 
         {{#if chat.activeChannel}}
-          {{#link-to
-            infoTabRoute
-            chat.activeChannel.id
-            chat.activeChannel.title
-            class="topic-chat-drawer-header__title"
-          }}
-            <div class="topic-chat-drawer-header__top-line">
-              {{#chat-channel-title channel=chat.activeChannel}}
-                {{#if (and (not expanded) unreadCount)}}
-                  <span class="chat-unread-count">{{unreadCount}}</span>
-                {{/if}}
-              {{/chat-channel-title}}
-            </div>
-
-            {{#if displayMembers}}
```

Comment:
The `displayMembers` computed property in the JS must be removed as well.

Reply:
there are internal cascades with that, I guess we can remove it but we would need to redo the backing cascade of dependencies in the model

Reply:
Oh sorry didn't see that, nevermind then